hack/ci-deploy.sh

```diff
@@ -15,32 +15,35 @@ if [[ -z ${1+x} ]]; then
     exit 1
 fi
 
-ARTIFACTS=${ARTIFACTS:-$( mktemp -d )}
+if [ -z "${DEPLOY_DIR+x}" ]; then
+  ARTIFACTS=${ARTIFACTS:-$( mktemp -d )}
```

Comment:
this shouldn't be conditioned on the deploy_dir

Reply:
agreed, fixed